macros/drop_schemas.sql

```diff
@@ -0,0 +1,31 @@
+{% macro drop_schemas(drop_target_schema=true) %}
```

Comment:
i wonder if you can make this more flexible by adding a `fuzzy_schema_match` condition? 

Specifically for [this use case](https://github.com/fivetran/dbt_package_integrations/blob/main/macros/delete_integration_tests.sql). If a `fuzz_schema_match = 'LIKE '%integration_test%'` could be a param, this can allow us to leverage the fivetran utils package for this instead of having a custom macro in the integrations package.

Reply:
I guess it doesn't have to be fuzzy lol it could just be something like `schema_where_like` . but probably depends on the use cases.

Reply:
oooh okay so twoish ideas: 
- maybe a `schema_patterns` variable that by default is `[target.schema]` so you can drop multiple schema patterns. so in this case you would provide `['integration_tests', 'integrations_tests']`. i realize this isn't what you were focusing on above but realized that's also a thing 😅 or you'd have to run the macro twice with different targets
- i opted for only adding the `%` wildcard as a suffix since that's in line with how dbt does custom schemas (ie appends `_schema_suffx` to the target). but yeah maybe that's too limiting.. i wonder if we add the `schema_patterns` variable, should the user have to add the `%`'s to the patterns? so like in your example, you would provide `schema_patterns=['%integration_test%', '%integrations_test%']` 
  - however, the default behavior could still align with dbt and be `schema_pattern=[target.schema ~ '%']`

we could also have the user provide a whole `where` clause like you mentioned, which would offer the most flexibility but fewer guardrails

cc @fivetran-joemarkiewicz

Reply:
Yeah I think for the current use case for this macro the way @fivetran-jamie opted to build it out works great! I could see a future where we provide more flexibility. 

However, for the current implementation I would prefer we keep it as is so we don't run into any fuzzy matching issues within our own warehouses. Especially as this will be running constantly as we work on updates across our packages. If we see interest in people wanting to leverage this outside of the current use case, we should definitely consider expanding the flexibility of the macro.

Reply:
cool i'll make a FR to include fuzzy matching after we merge this in!